src/o365/graph/commands/schemaextension/schemaextension-set.ts

```diff
@@ -0,0 +1,247 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  owner: string;
+  description?: string;
+  status?: string;
+  targetTypes?: string;
+  properties?: string;
+}
+
+class GraphSchemaExtensionSet extends GraphCommand {
+  public get name(): string {
+    return `${commands.SCHEMAEXTENSION_SET}`;
+  }
+
+  public get description(): string {
+    return 'Updates a Microsoft Graph schema extension';
+  }
+
```

Comment:
I added with only `status` and `targetTypes` which are the only standard parameters, all other are "user-defined" according to me, what do you think ?

Reply:
Looking at how options are defined in the command, description, status, targetTypes and properties are optional so we should track which of them are being used. Since the status is an enum, we could track the actual value.